### PR TITLE
paper rough draft

### PR DESCRIPTION
**Pull request by jodreen**

### CODE REVIEW

File: paper/report.tex [hunk 1]
```diff
@@ -20,70 +20,202 @@
...
-(RSA). To do so, we will create dissimilarity matrices for 18 individuals using
-the same searchlight mapping approach that they used (Subjects 4 and 10 were not
-included due to missing data). Doing so will capture 2nd-order isomorphisms in
-response patterns. Lastly, to access statistical significance, we will transform
-the representational consistency map into percent rank with respect to the total
-distribution of the DSM correlations. We'll calculate the mean correlation
-coefficient and compare our value to theirs.
+The main purpose of the original study was to examine properties of brain
+response patterns that are supposedly common when people are exposed to audio
+and movie simulation. We intend to replicate their experiment using the data
```
Comment (AlonDaks) [inline paper/report.tex]: simulation should be stimulation

File: paper/report.tex [hunk 2]
```diff
@@ -20,70 +20,202 @@
...
 
 \section{Methods}
+
+\subsection{Extracting Data}
+
+We will not be using all of the data. The data
+files are very large: for each subject, the download size is approximately 16
+GB, and that does not even include unzipping all the file inside. Given that
+we have twenty subjects, downloading all of this could be a paper in it of
+itself. and since the focus of this class is mainly fMRI data, we decided to
```
Comment (AlonDaks) [inline paper/report.tex]: there should be no period after itself.

File: paper/report.tex [hunk 3]
```diff
@@ -20,70 +20,202 @@
...
+ignore everything else. However, the fMRI data is still not of a trivial size
+because three versions of the data is included: the raw data, the linear
```
Comment (AlonDaks) [inline paper/report.tex]: is included should be *are* included

File: paper/report.tex [hunk 4]
```diff
@@ -20,70 +20,202 @@
...
+downloading a different quarter of the overall dataset, and then locally
+transferring the remaining three-quarters of the data from our hard drives.
+
+\subsection{Analysis}
+
+After perusing all the stimulus related data (e.g. data
+annotating each Forest Gump scene and when a subject was exposed to that
+scene), we decided to guide part of our analysis to see if we could summarize
+any interesting trends in fMRI response with respect to these features.
+Additionally, by investigating a single subject for our initial analysis  and
```
Comment (AlonDaks) [inline paper/report.tex]: extra space between "analysis  and"

File: paper/report.tex [hunk 5]
```diff
@@ -20,70 +20,202 @@
...
+scope to approximately 5 GBs from the roughly 320 GBs that were published with
+the paper. With the data reduced to a manageable state, we have successfully
+overcome this obstacle. Finally, by introducing a data\_path.json file to
+reference where each raw data file is located, we have designed a clean schema
+for our python scripts to find and load data in the project repository.
+
+We are deviating from the original data analysis. The paper used the raw data
+which varied largely with every subject, and processed that by standardizing
+among twenty of them with both linear alignments and non-linear alignments. We
+decided that we would just use the already processed linearly aligned data
```
Comment (AlonDaks) [inline paper/report.tex]: replace "just" with "simply"

File: paper/report.tex [hunk 6]
```diff
@@ -20,70 +20,202 @@
...
+since they already provided it, and we would not be able to do it as well as
+them.
+
+Our initial plan is to first just conduct some basic exploratory data analysis
```
Comment (AlonDaks) [inline paper/report.tex]: remove word "just"

File: paper/report.tex [hunk 7]
```diff
@@ -20,70 +20,202 @@
...
+a csv file of the scenes in the movie with a time-stamp, a brief scene
+description, whether it takes place in day or night, and if it is inside or
+outside. We will try to see if there is anything interesting in the data based
+on these characteristics of the movie scenes.
+
+After determining which points in time were during the day and which were
+during the night, and which were inside and which were outside, we can perform
+a t-test to determine if the signal is significantly different between each of
+the two groups for each voxel.  We will perform a multiple comparisons test to
+correct for the number of t-tests we will be running.   We will also model
```
Comment (AlonDaks) [inline paper/report.tex]: remove extra spaces between sentences

File: paper/report.tex [hunk 8]
```diff
@@ -20,70 +20,202 @@
...
+each of those voxels that are statistically significant to see what their time
+courses look like.
+
+We have a few ideas as to what questions to ask, but are in the process of
+looking at other ideas.  There is an entire website dedicated to studying this
```
Comment (AlonDaks) [inline paper/report.tex]: remove extra spaces between sentences

File: paper/report.tex [hunk 9]
```diff
@@ -20,70 +20,202 @@
...
+itself. Since the focus of this class is mainly fMRI data, we decided to
+ignore everything else. However, the fMRI data is still not of a trivial size
+because three versions of the data are included: the raw data, the linear
+alignment, and the non-linear alignment. We will only be choosing one of them
+(the one linearly aligned) because using all of them would just be redundant.
+
+We will be only working with the first subject for now for testing purposes,
+but if time allows, we will run our code on a couple of other subjects for
+comparison testing. And to ensure speedy access to other subjects' datasets,
+our strategy for getting all the data entails each group member spending
```
Comment (lisaannyu) [inline paper/report.tex]: remove spending (at the end of the sentence)

File: paper/report.tex [hunk 10]
```diff
@@ -20,70 +20,202 @@
...
-from what people experience in everyday life. This study sought to create
-a dataset of naturally occurring brain states by exposing participants
-to a more complex stimulus, the audio description of Forrest Gump. This particular
-audio description allows for the study of auditory attention and cognition, 
-language and music perception, and retrieval of explicit memory without the
-effect of visual imagery. In addition, this dataset uses inter-individual 
-synchronicity to study responses to complex processing. Originally,
+\maketitle
+
+\abstract{Most fMRI studies use highly simplified stimulus that are vastly
```
Comment (lisaannyu) [inline paper/report.tex]: stimuli

**Pull request merged**